application/modules/quotes/views/partial_itemlist_responsive.php

```diff
@@ -0,0 +1,318 @@
+<div class="row">
+	<div id="item_table" class="items table col-xs-12">

```

Comment:
Wrong indentation.
A lot of more lines not formatted accordingly (Look at https://github.com/InvoicePlane/InvoicePlane/blob/master/CONTRIBUTING.md)